### DISCUSSION TITLE
The contact point can support custom option in webhook

### DISCUSSION THREAD

**Discussion by Fury76** [grafana/grafana#53805] [Alerting]
I hop the webhook can support custom fields in http body and the value can support the template. Then the function of webhook will be greatly expanded. People can access other receiver without development

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
